src/i18n/en.json

```diff
@@ -0,0 +1,138 @@
+    "upgrade_account_free_HTML": "<p>Upgrade your account and access detailed Reports on the behavior of users on your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email that Doppler has identified and how many don't. Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p><p>By joining any Paid Plan you can enjoy FOR FREE this feature. Limited time only. <a href=\"{dopplerBaseUrl}/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\">UPGRADE NOW.</a></p>",
+    "upgrade_account_free_title": "Analyze your Subscriber's behaviour and improve your strategy"
+  },
+  "reports_box": {
+    "to": "to",
+    "visits_description_with_email": "Total number of users who visited your Website and whose Email has been identified by Doppler. If a user entered several times, only one will be counted.",
+    "visits_description_without_emails": "Total number of users who visited your Website and whose Email has not been identified by Doppler. If a user entered several times, only one will be counted.",
+    "visits_with_email": "Users with emails",
+    "visits_without_emails": "Users without email"
+  },
+  "reports_filters": {
+    "all_pages": "All pages",
+    "description_HTML": "<p>Find out which are the most visited pages, how many of those visitors already have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement! If you have any doubts, press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p>",
+    "domain": "Domain",
+    "domain_not_verified": "Not verified domain",
+    "pages": "Pages",
+    "rank_time": "Time period",
+    "title": "Track users behavior, analize it and optimize your Marketing actions",
+    "verified_domain": "Verified domain",
+    "week_with_plural": "{weeksCount, plural, =0 {no weeks} one {# week}other {# weeks} }"
+  },
+  "reports_pageranking": {
+    "top_pages": "Most visited pages",
+    "top_pages_sub_head":"Sessions are the total number of registered visits during the period. If a user entered several times, all of them will be counted.",
+    "total_visits": "Sessions"
+
+  },
+  "reports_title": "Doppler | Reports",
+  "signup": {
+    "activate_account_instructions": "* By clicking on the button from the Email, you will activate your account and you will be ready to enjoy all the benefits of Doppler.",
+    "button_signup": "create your free account",
+    "check_inbox": "Check your inbox. You have an email!",
```

Comment:
`Email`

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
`Check your inbox. You have an Email!`